src/main/java/com/stevedutch/assignment14/web/MessageController.java

```diff
@@ -25,17 +25,20 @@ public class MessageController {
 	private MessageService messageService;
 	
 	@PostMapping("channel")
-	public void postMessageInclUser(@RequestBody String json) throws JsonMappingException, JsonProcessingException {
```

Comment:
- no need to have a return statement for a void method
- no need to receive the request body as String and use the object mapper to map to the wanted class, Spring does that automatically for you